test/functional/feature_asset_locks.py

```diff
@@ -515,7 +519,7 @@ def run_test(self):
         all_mn_rewards = platform_reward + owner_reward + operator_reward
         assert_equal(all_mn_rewards, bt['coinbasevalue'] * 3 // 4)  # 75/25 mn/miner reward split
         assert_equal(platform_reward, all_mn_rewards * 375 // 1000)  # 0.375 platform share
-        assert_equal(platform_reward, 25553999)
+        assert_equal(platform_reward, 29636590)
```

Comment:
somehow this test is not 100% deterministic....
run 1
```
TestFramework (INFO): Wait for mn_rr activation
TestFramework (INFO): height: 2855 credit: 897274252983
```
run 2
```
TestFramework (INFO): Wait for mn_rr activation
TestFramework (INFO): height: 2831 credit: 895835213008
```
and it affects `platform_reward` too so maybe we could rely on formulas only here?
```suggestion
```

Reply:
ok, so it happened only once for me and I can no longer reproduce it... nvm then